lambda/MAS/Models/DailyEmail.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MAS.Models
+{
+    public class DailyEmail
+    {
+        public List<Item> Items { get; set; }
+
+        public List<IGrouping<string, Item>> GroupedItems
+        {
+            get
+            {
+                return Items.GroupBy(x => x.EvidenceType.Title).ToList();
+            }
+        }
+
+
+        public string HTML
```

Comment:
Does this belong in Models?

Reply:
It makes sense to me to have a daily email model. I agree with you that HTML property may not belong in this class - but im unsure as I've seen business logic in models before and the HTML is after all just a string property returning a string interpretation of the rest of the class. The HTML properties logic could be moved into a separate service which perhaps returns a dynamic view. However, for this card I think what we currently have is ok. Perhaps in the branding card, we can look at this. What do you think?